packages/cli-kit/src/public/node/git.ts

```diff
@@ -63,6 +63,34 @@ export function createGitIgnore(directory: string, template: GitIgnoreTemplate):
   appendFileSync(filePath, fileContent)
 }
 
+/**
+ * Add an entry to an existing .gitignore file.
+ *
+ * If the .gitignore file doesn't exist, or if the entry is already present,
+ * no changes will be made.
+ *
+ * @param root - The directory containing the .gitignore file.
+ * @param entry - The entry to add to the .gitignore file.
+ */
+export function addToGitIgnore(root: string, entry: string): void {
+  const gitIgnorePath = joinPath(root, '.gitignore')
+
+  if (!fileExistsSync(gitIgnorePath)) {
```

Comment:
Just curious, why sync instead of async?

Reply:
Generally, file system operations are quite fast, so I tend to use synchronous functions for them to avoid the requirement of async for consumers. If this were an HTTP request, I'd lean towards using async, so consumers can choose to handle those operations with a better approach :)